manifests/base/alertmanager/alert_rules.yaml

```diff
@@ -31,4 +31,9 @@ data:
             clusterID: "{{ $labels.clusterID }}"
             PersistentVolumeClaim: "{{ $labels.persistentvolumeclaim }}"
             severity: warning            
-
+        - alert: CriticalPolicyReport
+          annotations:
+            summary: There is a policy report with severity of critical.
+            description: "The policy: {{ $labels.policy }} has a severity of critical"
+          expr: sum(acm_policyreport_info{severity="critical"}) by (cluster,category, namespace, pod, policy)
```

Comment:
there should be some operator in the expr? such as ">1"

Reply:
thx - added `> 0`